### ISSUE TITLE
Contrasto insufficiente per la modalità non selezionata tra “Scrivi” e “Anteprima”

### ISSUE THREAD

**Issue by martalucivero** [italia/graph-italia#136]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [x] Ho verificato e non esiste

### Cosa

Nello stato non selezionato, il contrasto tra il testo e lo sfondo non raggiunge il rapporto minimo richiesto dalle WCAG.
Inoltre, se la differenza tra stato selezionato e non selezionato viene comunicata anche attraverso il colore, è necessario assicurarsi che lo stato rimanga distinguibile anche per gli utenti che non percepiscono correttamente le differenze cromatiche.

### Perché

WCAG 1.4.3 – Contrast (Minimum) (Livello AA): il testo deve avere un contrasto sufficiente rispetto allo sfondo.
WCAG 1.4.11 – Non-text Contrast (Livello AA): gli elementi visivi necessari per identificare componenti e stati devono avere un contrasto sufficiente.
WCAG 1.4.1 – Use of Color (Livello A): il colore non deve essere l'unico mezzo utilizzato per comunicare informazioni o stati.

### Contesto

<img width="544" height="418" alt="Image" src="https://github.com/user-attachments/assets/6bf66bae-4ba4-4550-8f08-e1254480a164" />
